src/main/resources/features/product_detail_page/see_size_guide.feature

```diff
@@ -0,0 +1,9 @@
+Feature: See Product Size Guide
+  The user should be able to see a size guide for all categories
+  of products on the web shop in a modal window when he/she clicks 
+  on the size guide button in the pdp (product detail page) of any product.
```

Comment:
"on the size guide button in the product detail page of any product."

You introduce an abbreviation which you don't use and should not use.